services/models/istio_validation.go

```diff
@@ -1,51 +1,42 @@
 package models
 
-// IstioTypeValidations represents a set of IstioNameValidations grouped per Istio ObjectType.
-// It is possible that different object types have same name, but names per ObjectType are unique.
-type IstioTypeValidations map[string]*IstioNameValidations
+// IstioValidationKey is the key value composed of an Istio ObjectType and Name.
+type IstioValidationKey struct {
+	ObjectType string
+	Name       string
+}
 
-// IstioNameValidations represents a set of checks grouped per Istio object.
-// The key of the map represents the name of the Istio object.
-type IstioNameValidations map[string]*IstioValidation
+// IstioValidations represents a set of IstioValidation grouped by IstioValidationKey.
+type IstioValidations map[IstioValidationKey]*IstioValidation
```

Comment:
How is going to be marshalled this one in json ?
From UI perspective, we expected to have a 
``` { "objectType": { "name": { ... validation ...}}} ```
Using a struct as a key sounds good, but not sure if we are breaking then the existing model we used.

Reply:
Ack, I need to test it. I've just managed to deploy Kiali from master so I'll try to run my local changes now...